src/components/Footer/Footer.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Footer.scss';
+
+class Footer extends Component {
+  render() {
+    return (
+      <div className="footer">
+        <div className="footerLink">
+          <div className="contain">
+            <ul>
+              <li>
+                <Link to="/">개인정보 취급방침</Link>
+              </li>
+              <li>
+                <Link to="/">이용약관</Link>
+              </li>
+              <li>
+                <Link to="/">매장안내</Link>
+              </li>
+              <li>
+                <Link to="/">1:1문의</Link>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div className="footerInfo">
+          <div className="contain">
+            <div className="customer">
+              <h3>모트모트 고객센터</h3>
+              <h4>
+                카카오톡 플러스 친구 <span>@motemote</span>
+              </h4>
+              <p>
+                평일 10:00 ~ 18:00 (점심시간 13:00 ~ 14:00) 주말 및 공휴일은
+                <br></br>
+                휴무입니다.
+              </p>
+            </div>
+            <div className="compInfo">
+              <address>
+                <span>주식회사 모트모트</span> <span className="bar"></span>
```

Comment:
```suggestion
                <span>주식회사 모트모트</span> <span className="bar" />
```
slef-closing 적용해주세요~

Reply:
나머지 부분도 마찬가지입니다 태그 본문에 아무것도 없으면 셀프클로징 적용해주세요